ENDPOINTS.md

```diff
@@ -2045,9 +2066,36 @@ Sample Output
 
 ```py
 {
+
   "Tag": "Capital Asset Pricing Model (CAPM)",
   "Risk-free rate of return": 86,
   "Asset's sensitivity": 6,
   "Expected return of the market": 87,
   "Expected return on the asset": "92.0%"
-}
\ No newline at end of file
+}
+
+**GET** `/risk_assessment`
```

Comment:
This should be the post request from now on since we are using request validators, and when those are implemented we need to provide a request body which is only possible in POST request. In docs, its needed to be changed too.

Reply:
Changed the method.